### PR TITLE
Increase Blast Grenade Recipe Cost and Add Research Requirement

### PR DESCRIPTION
**Pull request by Plykiya**
## About the PR

Blast grenades have been added to the Tier 2 Basic Shuttle Armament research and turned into dynamic recipes at the autolathe and secfab.

Blast grenades have also had their current recipe cost tripled.

## Why / Balance

Blast grenades, otherwise known as china lake ammo, are rarely printed. But when they are, they are often printed in such large quantities that the person with the china lake or friendship is capable of bombarding the station nonstop for 20+ minutes or killing everyone on the station thrice over. This excessive amount of ammo does nothing but promote murderboning rather than any strategic usage of the ammo to achieve their goals. The price has been adjusted to reflect the amount of damage each grenade is capable of.

Besides that, it's been research gated to go along with the proper ship armaments that it is intended for in terms of NanoTrasen usage. Not sure why the secfab didn't have the blast grenade recipe before, but it does now. Maybe this will allow security to build their own combat shittle to fight nukies or something...

## Technical details

just yml stuff
I also deleted some recipes in the secfab that were duplicated for some reason and moved PortableRechargers up to where they should be alphabetically

**Changelog**

:cl:
- tweak: Blast grenades have had their manufacturing cost tripled.
- tweak: Blast grenades are now part of the Basic Shuttle Armament research and can be printed at the secfab.

### CODE REVIEW

**Comment by Plykiya** [comment]
fwiw I'm up for any other suggestions on how to make the cost more difficult
I find my current idea of tripling the cost kind of lame
have had suggestions to throw in plasma to the recipe

**Comment by K-Dynamic** [comment]
I feel like ship cannons should be firing their own ammo, not sharing with the china lake

And the china lake grenades themselves need a nerf but that's another story

**Comment by Plykiya** [comment]
> I feel like ship cannons should be firing their own ammo, not sharing with the china lake
> 
> And the china lake grenades themselves need a nerf but that's another story

considering that sec seems to be intended to craft the grenades to load into their ship guns, we can at least operate under the assumption the ammo I'm talking about right now will be used in ship guns

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -753,14 +755,6 @@
       - WeaponLaserCannon
       - WeaponLaserCarbine
       - WeaponXrayCannon
-      - PowerCageSmall
-      - PowerCageMedium
-      - PowerCageHigh
-      - ShuttleGunSvalinnMachineGunCircuitboard
-      - ShuttleGunPerforatorCircuitboard
-      - ShuttleGunFriendshipCircuitboard
-      - ShuttleGunDusterCircuitboard
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:762]: Did you intend to remove all of these?

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml (hunk 1 above)
Comment (Plykiya) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:762]: "I also deleted some recipes in the secfab that were duplicated for some reason and moved PortableRechargers up to where they should be alphabetically"

**Pull request merged**